clang/test/CXX/drs/dr18xx.cpp

```diff
@@ -206,13 +206,10 @@ namespace cwg1814 { // cwg1814: yes
 #endif
 }
 
-namespace cwg1815 { // cwg1815: no
+namespace cwg1815 { // cwg1815: yes
```

Comment:
A test for constant evaluation would be nice here too. Maybe:
```c++
struct C { const int &r = 0; };
constexpr C c = {};
static_assert(c.r == 0);
```
or
```c++
constexpr int f() {
  A a = {};
  return a.r;
}
static_assert(f() == 0);
```

Reply:
Agree 100%!